libbpf-tools/Makefile

```diff
@@ -70,6 +70,7 @@ APPS = \
 	numamove \
 	offcputime \
 	oomkill \
+	profile \
```

Comment:
This and others mean your working directory is not clean. Could you rebase on top of master branch?

Reply:
have done, thanks for your reminding!